lib/channel/src/lib.rs

```diff
@@ -53,6 +53,7 @@
     receiver_events: Event,
     sender_events: Event,
     queue: ConcurrentQueue<T>,
+    on_demand_count: AtomicUsize,
```

Comment:
Is this count only used for stats?

Reply:
Yes. I originally added it as part of debugging to figure out what was going on with the on_demand events. But figured I'd leave it so we can see how many on_demand receivers there are over time in the dashboard.

Reply:
I would expect the count to not change over a test. At most I would expect the number of "on demand receivers" to be however many endpoints are `on_demand: true` + n (n being the number of providers since the process for watching file changes has a copy of all the providers). If you have one endpoint which is `on_demand` which finishes before the rest of the test, you may see that number decrement. All that's to say I'm not sure how useful keeping track of that number would be.

Reply:
Thanks. I may leave it in for now just to see how it works as I run through testing this build/preview out.